src/css/profile/mobile/common/slider.less

```diff
@@ -39,6 +38,13 @@
 		transform-origin: 0 50% 0;
 	}
 
+	&.ui-slider-active {
+	    .ui-slider-handler {
+			width: 22 * @px_base;
+			height: 22 * @px_base;
+	    }
+	}		
```

Comment:
Mixed indents in whole section. Please use tabs only.

Reply:
fixed